Lombiq.Tests.UI/BasicOrchardFeaturesTesting/BasicFeaturesTestingUITestContextExtensions.cs

```diff
@@ -17,18 +18,16 @@ namespace Lombiq.Tests.UI.BasicOrchardFeaturesTesting;
 public static class BasicFeaturesTestingUITestContextExtensions
 {
     /// <summary>
-    /// <para>
     /// Tests all the basic Orchard features. At first sets up Orchard with the recipe with the specified <paramref
     /// name="setupRecipeId"/>.
-    /// </para>
-    /// <para>The test method assumes that the site is not set up.</para>
     /// </summary>
     /// <param name="setupRecipeId">The ID of the recipe to be used to set up the site.</param>
     /// <param name="customPageHeaderCheckAsync">
     /// The custom page header check logic to locate and/or check the header's text. This ultimately gets passed to
     /// TestContentOperationsAsync().
     /// </param>
-    /// <returns>The same <see cref="UITestContext"/> instance.</returns>
+    /// <returns>The instance passed to <paramref name="context"/>.</returns>
```

Comment:
I guess this is some leftover, because neither of the methods return the context now.